index.js

```diff
@@ -1,6 +1,11 @@
 import Twitter from 'twit';
+import Promise from 'pinkie-promise';
 
-export default function getTwitterInfo(tokens, username, cb) {
+export default function getTwitterInfo(tokens, username) {
   const client = new Twitter(tokens);
-  client.get('users/show', { screen_name: username }, cb);
+
+  return new Promise((resolve, reject) => {
+    const handler = (err, res) => { err ? reject(err) : resolve(res) };
+    client.get('users/show', { screen_name: username }, handler);
+  });
```

Comment:
me neither and its fine, so im going to publish v2, once i will figure out why builds are failing under 0.12 and 0.10